app_dart/lib/src/model/luci/buildbucket.dart

```diff
@@ -207,6 +207,164 @@ class GetBuildRequest extends JsonBody {
   }
 }
 
+/// A request for the GetBuilder RPC.
+@JsonSerializable(includeIfNull: false)
+class GetBuilderRequest extends JsonBody {
+  /// Creates a request for the GetBuild RPC.
+  const GetBuilderRequest({
+    this.builderId,
+  }) : assert(builderId != null);
+
+  /// Creates a [GetBuilderRequest] from JSON.
+  static GetBuilderRequest fromJson(Map<String, dynamic> json) => _$GetBuilderRequestFromJson(json);
```

Comment:
No fix or anything just haven't seen this one before: _$GetBuilderRequestFromJson(json). Interesting notation.

Reply:
This is part of the codegen (see package:json_serializable)